awx/ui/src/screens/Job/JobOutput/EmptyOutput.js

```diff
@@ -21,6 +33,26 @@ export default function EmptyOutput({
     icon = SearchIcon;
   } else if (isJobRunning) {
     title = t`Waiting for job output…`;
+  } else if (job.status === 'failed') {
+    title = t`This job failed and has not output.`;
+    message = (
+      <>
+        {t`Please relaunch the job or, for more information see the`}{' '}
+        <Link to={`/jobs/${typeSegment}/${id}/details`}>{t`details.`}</Link>{' '}
```

Comment:
I don't find the advice to relaunch the job to be particularly good advice. My real recommendation would be to check the server logs _only if_ there is no information in the details.